src/UI/Settings/Quality/Definition/QualityDefinitionItemView.js

```diff
@@ -66,11 +65,11 @@ var view = Marionette.ItemView.extend({
     _changeSize : function() {
         var minSize = this.model.get('minSize') || this.slider.min;
         var maxSize = this.model.get('maxSize') || null;
-
+        console.log(minSize);
```

Comment:
so embarrassed about this was trying to get the pull request in before I went out on Sunday morning and didn't check back over all the files in my commit .